store/settings.py

```diff
@@ -78,12 +78,24 @@
 # https://docs.djangoproject.com/en/3.2/ref/settings/#databases
 
 DATABASES = {
+
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+
+        'NAME': 'dbp73lagh23fcp',
+
+        'USER': 'ctletkunmikemk',
+
+        'PASSWORD': 'a4c182a15fdd46e6d28c138d10518bcb38a6e1d218dc40a1a4bcea9a2b9111e5',
+
+        'HOST': 'ec2-34-233-214-228.compute-1.amazonaws.com',
+
+        'PORT': '5432',
+
     }
```

Comment:
@Ubaydah these variables here should be loaded from env
```
DATABASES = {

     'default': {
          'ENGINE': 'django.db.backends.postgresql_psycopg2',
 
          'NAME': os.getenv("DB_NAME"),
 
          'USER': os.getenv("DB_USER"),
 
          'PASSWORD': os.getenv("DB_PASSWORD"),
 
          'HOST': os.getenv("DB_HOST"),
 
          'PORT': '5432',
 
      }
```

Reply:
The add those variables to your local environment.

you can open you env file inside Scripts, you'll see activate.bat file. you can add

set <var>=<value> 

or you use normal terminal to set the variables.